sipa/static/css/agdsn-bootstrap-social.css

```diff
@@ -0,0 +1,67 @@
+.btn-social-icon {
+    position: relative;
+    padding-left: 44px;
+    text-align: left;
+    write-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    height: 34px;
+    width: 34px;
+    padding: 0;
+
+    background-color: var(--btn-social-bg-color);
+    color: #ffffff;
+
+    border: none;
+}
+.btn-social-icon>:first-child{
```

Comment:
First, I think `:first-child` is prematurely specific. The `.btn-social-icon` elements all only contain their respective icon `span` and nothing else.

Second, I think it should be much simpler to use flexbox for centering, let me have a stab at it and I'll get back to you.

Reply:
Throw a `d-flex justify-content-center align-items-center` on all the `<div>`s containing the social links and then
```css
.btn-social-icon {
    height: 36px;
    width: 36px;
    padding: 0;
    background-color: var(--btn-social-bg-color);
    color: white;
    border: none;
}
.btn-social-icon > *{
    font-size: 1.6em;
}
.btn-social-icon > *::before {
    vertical-align: -1.5pt;
}
```
should suffice for the positioning.
The last thing is a slight optical correction (which bs-icons do themselves, but works to our disadvantage in this context).

Reply:
Unfortunately, the icons still seem to be top aligned.  :(

Reply:
> Unfortunately, the icons still seem to be top aligned. :(



> Unfortunately, the icons still seem to be top aligned. :(

What do you mean by "still top aligned"? You never claimed they were top aligned before, and I never saw them top aligned.

Reply:
Resolved offline (734e80c00f337fd6f4021690d83fbe0ed853fa96)